source/includes/administration/_billing_providers.md

```diff
@@ -106,6 +106,126 @@ Provider | Attributes | &nbsp;
+      "creditCards": "Visa, Mastercard, American Express, Discover"
+    },
+    "defaultForType": false,
+    "id": "8db7cd13-b309-4fb9-b022-1632dbd165d0",
+    "updatedDate": "2021-05-17T19:59:33.387477Z",
+    "type": "CREDIT_CARD",
+    "providerType": "chaseCreditCard"
+  }
+}
+```
+
+Required | &nbsp;
+---------- | -----------
+`name`<br/>*string* | The name of the billing provider object.
+`organization.id`<br/>*UUID* | UUID of the organization to which belongs the billing provider. This can only be a organization which is root or reseller.
+`type`<br/>*string* | The type of billing provider. Possible values are: `CREDIT_CARD`.
+`providerType`<br/>*string* | The provider for the associated type.
+`configurationAttributes`<br/>*Object* | The configuration attribute associated to the provider type. [See possible values](#administration-provider-type-configuration-attribute) 
+`configurationAttributes.creditCards`<br/>*string* | The type of credit card accepted. This is a comma separated field.
+
+<!-------------------- UPDATE IDPS -------------------->
+
+#### Update billing provider
+
+`PUT /billing_providers/:id`
+
+Update an existing identity provider.
+
+```shell
+# Updates an existing identity provider
+curl -X PUT "https://cloudmc_endpoint/rest/billing_providers/c84cfe41-929b-47c9-bde4-b55a10bd2774" \
+   -H "MC-Api-Key: your_api_key"
+```
+
+> Request body example:
+
+```js
+{
+  "name": "Chase Credit Card (Dev)",
+	"organization": {
+			"id": "23910576-d29f-4c14-b663-31d728ff49a5"
+	},
+	"type": "CREDIT_CARD",
+  "providerType": "chaseCreditCard",
+	"configurationAttributes": {
+		"creditCards": "Visa, Mastercard, American Express, Discover"
+	} 
+}
+```
+> The above command return JSON structured like this:
+
+```js
+{
+  "data": {
+    "createdDate": "2021-05-17T20:32:07Z",
+    "organization": {
+      "id": "23910576-d29f-4c14-b663-31d728ff49a5"
+	  },
+    "name": "Chase Credit Card (Dev)",
+    "configurationAttributes": {
+      "creditCards": "Visa, MasterCard, American Express"
+    },
+    "defaultForType": false,
+    "id": "c84cfe41-929b-47c9-bde4-b55a10bd2774",
+    "updatedDate": "2021-05-17T21:09:49Z",
+    "type": "CREDIT_CARD",
+    "providerType": "chaseCreditCard"
+  }
+}
+```
+
+Required | &nbsp;
+---------- | -----------
+`name`<br/>*string* | The name of the billing provider object.
+`organization.id`<br/>*UUID* | UUID of the organization to which belongs the billing provider. This can only be a organization which is root or reseller.
+`type`<br/>*string* | The type of billing provider. Possible values are: `CREDIT_CARD`.
+`providerType`<br/>*string* | The provider for the associated type.
+`configurationAttributes`<br/>*Object* | The configuration attribute associated to the provider type. [See possible values](#administration-provider-type-configuration-attribute) 
+`configurationAttributes.creditCards`<br/>*string* | The type of credit card accepted. This is a comma separated field.
```

Comment:
I would remove this and have it in the attribute section

Reply:
da4d7b6